vernac/pvernac.mli

```diff
@@ -25,7 +25,9 @@ module Vernac_ :
     val command : vernac_expr Entry.t
     val syntax : vernac_expr Entry.t
     val vernac_control : vernac_control Entry.t
+    val fix_definition : fixpoint_expr Entry.t
     val rec_definition : fixpoint_expr Entry.t
+      [@@deprecated "Deprecated in 8.13; use 'fix_definition' instead"]
```

Comment:
Why not just delete it?

Reply:
Global variables for nonterminals defined in `mli` files that were renamed were made equal to the new variable and deprecated to avoid breaking external plugins that may rely on those variables.

If it's easy to avoid breaking external plugins I will do it.